components/dialogs/AuthDialog.vue

```diff
@@ -421,9 +427,12 @@ export default {
         this.setUserNeedAuth(false);
         this.redirectToUserPage();
       } catch (err) {
-        console.error(err);
-        // TODO: Check error
-        // Assume it is 404
+        if (!err.response || err.response.status !== 404) {
+          this.currentTab = 'signInError';
+          return;
+        }
```

Comment:
same as above, `404` should be explicitly handle as a case

Reply:
Updated